Content.Server/StationEvents/Components/CargoGiftsRuleComponent.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Content.Server.StationEvents.Events;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+
+namespace Content.Server.StationEvents.Components;
+
+/// <summary>
+/// Used an event that gifts the station with certian cargo
+/// </summary>
+[RegisterComponent, Access(typeof(CargoGiftsRule))]
+public sealed class CargoGiftsRuleComponent : Component
+{
+    /// <summary>
+    /// What is being sent
+    /// </summary>
+    [DataField("descr"), ViewVariables(VVAccess.ReadWrite)]
+    public string Descr = "A bundle of gifts";
+
+    /// <summary>
+    /// Sender of the gifts
+    /// </summary>
+    [DataField("sender"), ViewVariables(VVAccess.ReadWrite)]
+    public string Sender = "NanoTrasen";
+
+    /// <summary>
+    /// Destination of the gifts (who they get sent to on the station)
+    /// </summary>
+    [DataField("careof"), ViewVariables(VVAccess.ReadWrite)]
+    public string Careof = "The Cargo Dept.";
```

Comment:
...Careof? I have no idea what that means.

Reply:
When you send a delivery to someone who can't receive it directly, you send it "care of" someone else. `CareOf` variable name perhaps?